pkg/operator/staticpod/controller/installer/installer_controller.go

```diff
@@ -372,26 +372,25 @@ func (c *InstallerController) manageInstallationPods(ctx context.Context, operat
 	return false, nil
 }
 
-func (c *InstallerController) updateRevisionStatus(ctx context.Context, operatorStatus *operatorv1.StaticPodOperatorStatus) error {
+func (c *InstallerController) updateRevisionStatus(ctx context.Context, operatorStatus *operatorv1.StaticPodOperatorStatus, logLevel operatorv1.LogLevel) error {
```

Comment:
having log level as something special here looks odd.

Reply:
yeah, I will make this more reasonable.

Reply:
having logLevel configurable is not necessary for my use case I hardcoded Normal (2)